src/app/shared/dialog/facet-entries/facet-entries.component.ts

```diff
@@ -224,6 +226,22 @@ export class FacetEntriesComponent implements OnDestroy, OnInit {
     );
   }
 
+  public getAccessibilityLabelEntry(entry: SdrFacetEntry): string {
+    const translatedString = this.translate.instant('SHARED.DIALOG.FACET_ENTRIES.ACCESSIBILITY.FILTER_BY');
+    return `${this.accessibilityFilterLabel = translatedString}-${this.getStringValue(entry)}`;  
+  }
+
+  public getAccessibilityLabelFacetRange(facet: Facet, entry: SdrFacetEntry): string {
+    const translatedString = this.translate.instant('SHARED.DIALOG.FACET_ENTRIES.ACCESSIBILITY.FILTER_BY');
+    return `${this.accessibilityFilterLabel = translatedString}-${this.getFacetRangeValue(facet, entry)}`;  
+  }
+
+  public getAccessibilityLabelDateYear(entry: SdrFacetEntry): string {
+    const translatedString = this.translate.instant('SHARED.DIALOG.FACET_ENTRIES.ACCESSIBILITY.FILTER_BY');
+    const formattedDate = new Date(entry.value).getUTCFullYear();
+    return `${translatedString}-${formattedDate}`;
+  }
+
```

Comment:
This seems more complicated than just using a pipe as was done before.
Is there a performance reason to use it this way?

I think the previous pipe approach is a lot cleaner.